package.json

```diff
@@ -14,7 +14,8 @@
     "prettier": "eslint '**/*.js' --fix",
     "e2e:build": "cd __e2e__/ && webpack",
     "e2e:start": "node __e2e__/dist/index.js",
-    "e2e:watch": "npm run e2e:build && nodemon __e2e__/dist/ --watch __e2e__/dist/"
+    "e2e:watch": "npm run e2e:build && nodemon __e2e__/dist/ --watch __e2e__/dist/",
+    "deployment:android": "cd android && fastlane travistest"
```

Comment:
`fastlane travistest` 

Is `travistest` supposed to be a lane? I can't find it

Reply:
Removed